setup.py

```diff
@@ -52,7 +52,6 @@
         'Record',
         'RestrictedPython',
         'zExceptions',
-        'ZODB3',
```

Comment:
Let's add `ZODB` and `BTrees` as an explicit dependency, since this product imports from them directly.